src/types/expected-errors/registerBlockBindingsSource.ts

```diff
@@ -0,0 +1,36 @@
+// @ts-expect-error Temporary registerBlockBindingsSource type error workaround for WordPress 6.7
+import { registerBlockBindingsSource as originalRegisterBlockBindingsSource } from '@wordpress/blocks';
+
+import type {
+	BlockEditorStoreActions,
+	BlockEditorStoreSelectors,
+	BlockEditorStoreDescriptor,
+} from '@wordpress/block-editor';
+
+interface GetValuesPayload< Context, Values > {
+	bindings: Values;
```

Comment:
this probably isn't right, but not sure what it should be. we can improve these once we start to use them